app/views/categories/_content_items.html.erb

```diff
@@ -1,7 +1,15 @@
 <ol class="category-detail__list">
-  <% contents.each do |item| %>
+  <% contents.take(6).each do |item| %>
     <li class="category-detail__list-item">
       <%= link_to item.title, item.path %>
     </li>
   <% end %>
 </ol>
+<ol class="collapsible-section category-detail__list category-detail__list--extended js-category-detail__list--extended">
+  <% (contents[6..-1] || []).each do |item| %>
```

Comment:
Aha, that makes sense. In that case, you could put this another way:

```ruby
Array(contents[6..-1]).each do |item|
```

Up to you — I find the latter clearer but others might feel differently.